core/src/avm2/globals/flash/system/MessageChannel.as

```diff
@@ -0,0 +1,9 @@
+package flash.system {
+    import flash.events.EventDispatcher;
+
+    public final class MessageChannel extends EventDispatcher {
+        public function MessageChannel() {
```

Comment:
This seems redundant

Reply:
What should it be replaced with? I think there's some constructor behavior, if we ever implement it